.chainerci/run-chainerci-tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -eux
+
+cat <<'EOM' >runtest.sh
+set -eux
+
+python3 -m pip install gast
+python3 -m pip install --pre cupy-cuda100==7.0.0a1
+
+CHAINER_BUILD_CHAINERX=1 CHAINERX_BUILD_CUDA=1 MAKEFLAGS=-j8 \
+    python3 -m pip -q install --no-cache-dir third_party/chainer[test]
+python3 -m pip install --no-cache-dir third_party/onnx-chainer[test-gpu]
+
+mkdir -p build
+cd build
+cmake .. \
+    -G Ninja \
+    -DCHAINER_COMPILER_ENABLE_CUDA=ON \
+    -DCHAINER_COMPILER_ENABLE_CUDNN=ON \
+    -DCHAINER_COMPILER_ENABLE_OPENCV=ON \
+    -DCHAINER_COMPILER_ENABLE_PYTHON=ON \
+    -DCHAINER_COMPILER_NGRAPH_DIR=$HOME/ngraph_dist \
+    -DPYTHON_EXECUTABLE=/usr/bin/python3 \
+    -DCHAINER_COMPILER_ENABLE_TVM=ON \
+    -DCHAINERX_BUILD_CUDA=ON \
+    -DCHAINERX_BUILD_PYTHON=ON \
+    && \
+    ninja
+
+make large_tests
+
+make test
+cd ..
+
+python3 scripts/runtests.py -G --ngraph
```

Comment:
FYI, setting `-G` and `--ngraph` at once does not make much sense (I don't think ngraph op can handle cuda ndarray). I think CI should run runtests.py several times for each backend.

Reply:
I split those conditions, thx

```
python3 scripts/runtests.py -g
python3 scripts/runtests.py --ngraph
```